PLUGIN_MIGRATION.md

```diff
@@ -31,12 +31,18 @@ The `SDKClient` provides two (eventually three) client options.
 
 The [Java Client for OpenSearch](https://github.com/opensearch-project/opensearch-java) (`OpenSearchClient`) will be supported with both synchronous and asynchronous clients, and is actively developed along with other language clients and should be used whenever possible. These clients do have significant implementation differences compared to the existing `Client` interface implemented by plugins.
 
+## Change Plugin and OpenSearch TransportAction implementations
+
 The `SDKRestClient` provides wrapper methods matching the `Client` API (but not implementing it), implemented internally with the (soon to be deprecated) `RestHighLevelClient`.  While this speeds migration efforts, it should be considered a temporary "bridge" with follow up migration efforts to the `OpenSearchClient` planned.
  - While the class names and method parameters are the same, the `Request` and `Response` classes are often in different packages. In most cases, other than changing `import` statements, no additional code changes are required. In a few cases, there are minor changes required to interface with the new response class API.
 
 The `client.execute(action, request, responseListener)` method is implemented on the SDKClient.
 
-Change the transport action inheritance from HandledTransportAction to directly inherit from `TransportAction`.
+For TransportActions internal to the plugin (registered with `getActions()`), change the transport action inheritance from HandledTransportAction to directly inherit from `TransportAction`.
+
+TransportActions on OpenSearch are not accessible to extensions, and will need to be replaced with functionality from either a client or some other functionality directly provided by the Extensions SDK.  A few examples of the types of changes needed include:
```

Comment:
```suggestion
TransportActions on OpenSearch are not accessible to extensions, and will need to be replaced with functionality from either a java client or some other functionality directly provided by the Extensions SDK.  A few examples of the types of changes needed include:
```

Reply:
Hmm, I agree we need more than "a client" but the "java client" isn't much better. I'll think about how to improve this.